packages/react-core/src/components/Label/examples/Label.md

```diff
@@ -6,6 +6,7 @@ propComponents: ['Label']
 ---
 
 import InfoCircleIcon from '@patternfly/react-icons/dist/js/icons/info-circle-icon';
+import { Link } from '@reach/router';
```

Comment:
we don't actually have this in our dependencies, maybe better to create a fake component as an example
`const FakeRouterLink = ({ to, children, ...props }) => <a href={to} {...props}>{children}</a>;`

Reply:
well somehow it works, not exactly sure how @redallen. I guess you can leave it!

Reply:
It works because the theme uses that router and it's globally available there.